src/scss/generic/_body.scss

```diff
@@ -1,14 +1,17 @@
 body {
-  @include media(medium) {
-    margin: 0 0 map-deep-get($footerSize, mini, size);
-  }
-
-  margin: 0 0 map-deep-get($footerSize, large, size);
   font: {
     size: 16px;
     family: font-family('Lato');
     weight: 400;
   };
   line-height: 1.3;
   color: get-color(blackNeutral);
+
+  .sticky & {
+    @include media(medium) {
+      margin: 0 0 map-deep-get($footerSize, mini, size);
```

Comment:
You could only overwrite just margin-bottom here, default margin for (top, right and left) is 0.